testdata/go/v4-alpha/memcached-operator/test/e2e/e2e_test.go

```diff
@@ -76,6 +86,26 @@ var _ = Describe("memcached", func() {
 				"pod-security.kubernetes.io/enforce=restricted")
 			_, err = utils.Run(cmd)
 			Expect(err).To(Not(HaveOccurred()))
+
+			By("uncommenting all sections with 'monitoring' to enable operator custom metrics")
+			cmd = exec.Command("sed", "-i",
+				`s/\/\/"github.com\/example\/memcached-operator\/monitoring"/"github.com\/example\/memcached-operator\/monitoring"/g`,
```

Comment:
ARe you using Golang IDEA? 
If so, see if that has a plugin to work with strings
Note that if we use conts as fragments the code is much more readable and easy to keep maintained

Example: 
https://github.com/operator-framework/operator-sdk/blob/280335593c76c1fb14ce96100938f140ddb42659/hack/generate/samples/internal/ansible/advanced_molecule.go#L129-L157

Could we update it?

Reply:
Done